### PR TITLE
feat/view-25 - Simple rating component + movies/:id route

### PR DESCRIPTION
**Pull request by Cygap**
# Description: added route movies/:id to app.js (allows to test frontend components without modifying app.js further, added empty MovieView component to contain all the frontend of the View page, added simple rating component from MUI

# Jira task link: https://hackyourfuture-dk.atlassian.net/browse/VIEW-25

# Screenshots (if applicable) 
image

### CODE REVIEW

**Review by shpomp — CHANGES_REQUESTED** [review]
No package updates in PRs, unless coordinated and agreed 🙏🏽

**Comment by Cygap** [comment]
Sorry for that. These were traces of me fixing a problem with linters... Now everything should be ok. Can you please check?

**Review by shpomp — CHANGES_REQUESTED** [review]
By all means looks good - nice and simple! ⭐ 

However, it is not finished and final as per the design, and neither Jira not PR description indicates that more work will happen on it as the next step.

File: packages/client/src/App.js [hunk 1]
```diff
@@ -3,13 +3,16 @@ import { BrowserRouter as Router, Routes, Route } from 'react-router-dom';
...
 import { PageNotFound } from './containers/PageNotFound/PageNotFound.Container';
+import MovieView from './components/MovieView/MovieView.component';
 
 function App() {
   return (
     <div className="app">
       <Router>
         <Routes>
           <Route path="/" element={<LandingPage />} />
+          {/* Added movies/:id path to be able to test frontend components for the Movie View page */}
```
[the remark this answers is not in the document]
Comment (Cygap) [reply packages/client/src/App.js:14]: Removed comments... and also added more reasonable description to Jira task

**Comment by shpomp** [comment]
There is a lot of confusion here still. 🥺 

The user story title is unclear. 
The user story description that this task is under now is "As a developer, I want to see the reviews of the movies in the database". So this task is unrelated to the story and should not be there. 
There is another task that is also unrelated to the story.
image

When the task gets to where it needs to be, the Jira tasks situation should reflect the totality of the user story. 
The end result of the story being fulfilled is the right design of the rating component - which should be a task in itself, that could even be already linked here showing that the styling is not final, but will be finalized in this sprint. 
In that way, anybody looking at it would know that this is not final and there is a task planned in this sprint to come back and adjust the styles. 

@Cygap 
and @Aditi-Sawardekar  as the Agile Agent ⭐

**Comment by Cygap** [comment]
We have moved it to a more appropriate place in Jira already, removed comments

**Pull request merged**